lib/visitor-keys.js

```diff
@@ -7,36 +24,27 @@ const KEYS = {
         "left",
         "right"
     ],
-    ArrayExpression: [
-        "elements"
-    ],
-    ArrayPattern: [
-        "elements"
-    ],
-    ArrowFunctionExpression: [
-        "params",
-        "body"
+    AssignmentProperty: [
+        "key",
+        "value"
     ],
     AwaitExpression: [
         "argument"
     ],
-    BlockStatement: [
-        "body"
-    ],
+    BigIntLiteral: [],
     BinaryExpression: [
         "left",
         "right"
     ],
+    BlockStatement: [
+        "body"
+    ],
     BreakStatement: [
         "label"
     ],
-    CallExpression: [
-        "callee",
-        "arguments"
-    ],
     CatchClause: [
-        "param",
-        "body"
+        "body",
+        "param"
     ],
```

Comment:
These changes in the order of the keys would be problematic, as ESLint rules (and other things, like code path analysis https://github.com/eslint/eslint/issues/13030#issuecomment-597740557) expect certain traversal order.

Reply:
Ok, thanks. See my reply above.